graphql_compiler/tests/conftest.py

```diff
@@ -24,7 +24,6 @@ def init_snapshot_graph_client():
     return _init_graph_client(load_schema, generate_orient_snapshot_data)
 
 
-@pytest.fixture(scope='session')
```

Comment:
I had to remove this as a fixture to fix the test.

Reply:
What's the reasoning for removing this? Doesn't this mean that the client will get initialized for every single test class annotated with the other fixture?

Also, I'm not a fan of changing test setup together with changing documentation.

Reply:
So this is a bug fix. The README.md example uses this as a function but it's a fixture. 

How about I also create an Init_integration_graph_client_fixture function to avoid initializing the client every single test case.